### PR TITLE
Fix schedule routing issue

### PR DESCRIPTION
**Pull request by matoval**
Jira - https://issues.redhat.com/browse/AAP-21454

Fix schedule create routing issue

### CODE REVIEW

File: frontend/awx/views/schedules/components/ScheduleInputs.tsx [hunk 1]
```diff
@@ -67,10 +67,12 @@ export function ScheduleInputs(props: {
         return;
       }
 
+      const pathnameSplit = pathname.split('/');
+      const resourceType = pathnameSplit[1] === 'projects' ? 'projects' : pathnameSplit[2];
       try {
         const response = await requestGet<
           Project | JobTemplate | WorkflowJobTemplate | InventorySource
-        >(`${resourceEndPoints[pathname.split('/')[2]]}${params?.id}/`);
+        >(`${resourceEndPoints[resourceType]}${params?.id}/`);
```
[the remark this answers is not in the document]
Comment (matoval) [reply frontend/awx/views/schedules/components/ScheduleInputs.tsx:75]: Which schedule list Create Schedule didn't work for you? I'm not able to recreate this bug.

[the remark this answers is not in the document]
Comment (matoval) [reply frontend/awx/views/schedules/components/ScheduleInputs.tsx:75]: The issue is we shouldn't have the `Create schedule` button on `/schedules` list.

[the remark this answers is not in the document]
Comment (matoval) [reply frontend/awx/views/schedules/components/ScheduleInputs.tsx:75]: Fixed in latest push

**Review by AlexSCorey — APPROVED** [review]
I've pulled this down and tested it out.  Everything looks great!

**Pull request merged**